src/routes/_components/DynamicPageBanner.html

```diff
@@ -34,19 +37,25 @@ <h1 class="dynamic-page-title" aria-label={ariaTitle}>{title}</h1>
     text-overflow: ellipsis;
   }
   .dynamic-page-go-back {
-    font-size: 1.3em;
+    display: inline-flex;
+    align-items: center;
+    justify-self: flex-end;
+    font-size: 1.2857142857142858em;
```

Comment:
Was this `font-size` generated by a tool or something? Same question about the `0.6666666` below. :slightly_smiling_face:

Reply:
To get the alignment on a pixel grid I chose to calculate this so it rounds to the nearest pixel.